sdk/export/metric/aggregation/temporality.go

```diff
@@ -0,0 +1,119 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:generate stringer -type=Temporality
+
+package aggregation // import "go.opentelemetry.io/otel/sdk/export/metric/aggregation"
+
+import (
+	"go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/metric/sdkapi"
+)
+
+// Temporality indicates the kind of data exported by an exporter.
+// These bits may be OR-d together when multiple exporters are in use.
+type Temporality int
+
+const (
+	// CumulativeTemporality indicates that an Exporter expects a
+	// Cumulative Aggregation.
+	CumulativeTemporality Temporality = 1
+
+	// DeltaTemporality indicates that an Exporter expects a
+	// Delta Aggregation.
+	DeltaTemporality Temporality = 2
+)
+
+// Includes tests whether `kind` includes a specific kind of
+// exporter.
+func (kind Temporality) Includes(has Temporality) bool {
+	return kind&has != 0
```

Comment:
Can we rename the receiver variable to match the type name? i.e.

```suggestion
// Includes returns if t includes support for other temporality.
func (t Temporality) Includes(other Temporality) bool {
	return t&other != 0
```